src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const MEDIUM_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const MEDIUM_TERM_DISCOUNT = 20;
+  const BASE_PRICE = 40;
+
+  if (days >= LONG_TERM) {
+    return days * BASE_PRICE - LONG_TERM_DISCOUNT;
```

Comment:
days * BASE_PRICE, this operation repeating 3 times, better also create const for this

Reply:
Thank you, fixed